python-avd/pyavd/_eos_designs/structured_config/network_services/utils.py

```diff
@@ -199,6 +199,10 @@ def _configure_bgp_mlag_peer_group(self: AvdStructuredConfigNetworkServices) ->
 
         return False
 
+    ################
```

Comment:
Maybe a separate mixin?

Reply:
yeah probably a btter plan it starts to grow a bit big